docs/getting-started.md

```diff
@@ -300,8 +302,10 @@ Supported Boards:
 - nuc11tnbi5
 - cfl-k700-i7
 - whl-ipc-i5
```

Comment:
Update here too

Reply:
should I add nuc11tnbv7 after it is verified? this is Jing Hui's nuc11.

Reply:
No